packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -84,6 +84,12 @@ export class VaTextInput {
     | 'text'
     | 'url';
 
+  /**
+   * The step attribute is a number, or the string 'any', that specifies the granularity of the value. For example: `<va-text-input type="number" step=".1"/>` enables float/decimal values to be valid and increment by one-tenth. <br/>
+   * Defaults to 1 for every field type except for time and datetime-local which default to 60 (seconds). View more information [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step)
```

Comment:
Nice! Markdown supported! To make this link more accessible, can have the link text be more descriptive instead of just "here"? Maybe just "MDN" is fine?